src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+//로또 1장
+public class Lotto {
+
+    private final List<LottoNumber> numbers;
+
+    public Lotto(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    public static Lotto generateAutoLotto() {
+        List<Integer> pickedNums = IntStream.range(1, 46)
```

Comment:
로또의 번호는 1에서 45 까지에요.

그런데 46이 들어가면 혼란이 있지 않을까요?

```suggestion
        List<Integer> pickedNums = IntStream.rangeClosed(1, 45)
```

Reply:
이런문법이 있었군요... 확실히 범위를 확인할 수 있을 것 같습니다. 앞으로는 `rangeClosed`를 사용해 범위 혼란을 줄여보겠습니다!